crates/ra_hir/src/code_model_api.rs

```diff
@@ -352,6 +352,20 @@ impl Function {
     pub fn generic_params(&self, db: &impl HirDatabase) -> Arc<GenericParams> {
         db.generic_params(self.def_id)
     }
+
+    pub fn docs(&self, db: &impl HirDatabase) -> Option<String> {
+        let def_loc = self.def_id.loc(db);
+        let syntax = db.file_item(def_loc.source_item_id);
+        let fn_def = ast::FnDef::cast(&syntax).expect("fn def should point to FnDef node");
+
+        // doc_comment_text unconditionally returns a String
+        let comments = fn_def.doc_comment_text();
```

Comment:
This probably means we should fix `doc_comment_text` :)

Reply:
Agreed. I will do that in another PR.